DependencyInjection/CmfSeoExtension.php

```diff
@@ -242,5 +250,9 @@ private function loadSitemapHandling($config, XmlFileLoader $loader, ContainerBu
             $this->getAlias().'.sitemap.default_change_frequency',
             $config['default_change_frequency']
         );
+
+        if ($this->isConfigEnabled($container, $persistence['phpcr'])) {
+            $loader->load('phpcr-sitemap.xml');
+        }
```

Comment:
moving this here, as otherwise we load this bit even when sitemaps are not enabled, leading to missing default_change_frequency parameter.